api/src/utils/sanitize.js

```diff
@@ -0,0 +1,11 @@
+import sanitizeHtml from 'sanitize-html';
+
+export default function sanitize(dirty) {
```

Comment:
If we have a `sanitize` in utilities, we should also replace usage of https://github.com/GetStream/Winds/blob/master/api/src/parsers/feed.js#L32 with our new `sanitize` utility

Reply:
I considered that at the first commit, https://github.com/GetStream/Winds/pull/526/commits/3ce1c72387a6ab9a747e66a7522888473817bf5f#diff-cf0dc246fefcb6a2c2e8be8b7a655a4c

however, feed class only sanitize for the summary and I suppose it shouldn't contain links or some specific tags that we might support in the future for the content.